config/dev.secret.example.exs

```diff
@@ -4,3 +4,11 @@ use Mix.Config
 # config :movie_wager_api, MovieWagerApi.Repo,
 #   username: "postgres",
 #   password: "postgres"
+
+# Replace with your twitter app credentials
```

Comment:
ah, you're ahead of me. This is enough info, I don't think it needs more comments.